src/vs/workbench/contrib/preferences/browser/preferences.contribution.ts

```diff
@@ -508,8 +508,8 @@ MenuRegistry.appendMenuItem(MenuId.CommandPalette, {
 	when: WorkbenchStateContext.isEqualTo('workspace')
 });
 
-CommandsRegistry.registerCommand(OpenWorkspaceSettingsAction.ID, serviceAccessor => {
-	serviceAccessor.get(IInstantiationService).createInstance(OpenWorkspaceSettingsAction, OpenWorkspaceSettingsAction.ID, OpenWorkspaceSettingsAction.LABEL).run();
+CommandsRegistry.registerCommand(OpenWorkspaceSettingsAction.ID, (serviceAccessor, args) => {
+	serviceAccessor.get(IInstantiationService).createInstance(OpenWorkspaceSettingsAction, OpenWorkspaceSettingsAction.ID, OpenWorkspaceSettingsAction.LABEL).run(args);
```

Comment:
I think we already support this for settings.

https://github.com/microsoft/vscode/blob/6eaf748366a48a5b6f9645cfa5e2257f51a027a6/src/vs/workbench/contrib/preferences/browser/preferences.contribution.ts#L230-L240

So not sure if you want to do this for all commands. @roblourens FYI